### PR TITLE
AV-210878: Increase API timeout to 200 seconds

### PR DESCRIPTION
**Pull request by akshayhavile**

### CODE REVIEW

File: pkg/utils/avi_rest_utils.go [hunk 1]
```diff
@@ -45,6 +46,7 @@ func NewAviRestClientPool(num uint32, api_ep, username,
 	options := []func(*session.AviSession) error{
 		session.SetNoControllerStatusCheck,
 		session.SetTransport(transport),
+		session.SetTimeout(200 * time.Second),
```
Comment (abhishekbsingh) [inline pkg/utils/avi_rest_utils.go:49]: nit: increasing the timeout here should be good enough as other clients do not get used in SSL cert creation in Avi.

Comment (akshayhavile) [reply pkg/utils/avi_rest_utils.go:49]: Currently, we only know sslcert is taking time. To secure other API calls and paths which is not covered by this flow, we are fending it with this timeout. There are bunch of places, we directly create session object and connect to controller.

**Comment by akshayhavile** [comment]
build ako

**Review by abhishekbsingh — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Review by DixitAakash — APPROVED** [review]

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Comment by akshayhavile** [comment]
build ako

**Pull request merged**